src/som/primitives/SystemPrimitives.java

```diff
@@ -104,28 +117,43 @@ public void invoke(final Frame frame, final Interpreter interpreter) {
     startTime = startMicroTime / 1000L;
     installInstancePrimitive(new SPrimitive("time", universe) {
 
-      public void invoke(final Frame frame, final Interpreter interpreter) {
-        frame.pop(); // ignore
+      public void invoke(final VirtualFrame truffleFrame,
+          final Interpreter interpreter) throws FrameSlotTypeException {
+
+        StackUtils.pop(truffleFrame); // ignore TODO- why?
```

Comment:
This is a unary primitive on the system object. We don't need the system object, so, we simply pop it off the stack.